.github/workflows/nightly-benchmark.yml

```
@@ -350,6 +350,7 @@ jobs:
           - { id: meta-llama/Llama-4-Maverick-17B-128E-Instruct-FP8, slug: meta-llama-Llama-4-Maverick-17B-128E-Instruct-FP8, test_class: TestNightlyLlama4MaverickSingle }
           # Pausing MiniMax M2 nightly benchmark
           # - { id: minimaxai/minimax-m2, slug: minimaxai-minimax-m2, test_class: TestNightlyMinimaxM2Single }
+          - { id: mistralai/Devstral-2-123B-Instruct-2512, slug: mistralai-Devstral-2-123B-Instruct-2512, test_class: TestNightlyDevstral2Single }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)